### PR TITLE
New Package: mopidy-mpris-1.3.1

### PR DESCRIPTION
**Pull request by Pizza-Boy**

### CODE REVIEW

File: srcpkgs/mopidy-mpris/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#Template for building the 'Mopidy-MPRIS' package
+pkgname=mopidy-mpris
+version=1.3.1
+revision=1
+wrksrc="mopidy-mpris-$version"
```
Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: This line is not needed. By default `wrksrc="${pkgname}-${version}"`

File: srcpkgs/mopidy-mpris/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#Template for building the 'Mopidy-MPRIS' package
```
Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: Convention: `# Template file for 'mopidy-mpris'`

**Comment by Pizza-Boy** [comment]
Suggested changes should have been committed

File: srcpkgs/mopidy-mpris/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#Template for building the 'Mopidy-MPRIS'
+pkgname=mopidy-mpris
+version=1.3.1
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module=mopidy-mpris
+hostmakedepends="python-setuptools"
+depends="python-dbus mopidy python-pykka python"
```
Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: `python` is pulled in by `python-pykka` and `python-dbus` already. `python-pykka` is pulled in by `mopidy`. Thus, there should only be two packages on this list.

**Comment by Pizza-Boy** [comment]
Corrected

**Comment by Vaelatern** [comment]
You somehow reverted the commit at the top ;)

File: srcpkgs/mopidy-mpris/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#Template for building the 'Mopidy-MPRIS'
+pkgname=mopidy-mpris
+version=1.3.1
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module=mopidy-mpris
```
Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: `pycompile_module="mopidy_mpris"`

**Comment by Pizza-Boy** [comment]
Made a small type correction to the `wrksrc` line.

That `wrksrc` line is actually necessary because the default package downloaded from PyPI has some funky capitalization and the directory built in `builddir` is titled "Mopidy-MPRIS-${version}" instead of "mopidy-mpris-${version}".

I think I originally had two `wrksrc` definitions floating around the file due to general bad scripting/disorganization on my part, my bad.

**Comment by Pizza-Boy** [comment]
Corrected the "pycompile_module" line

**Comment by Vaelatern** [comment]
Waiting for the travis build to complete sucessfully.

File: srcpkgs/mopidy-mpris/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
+#Template for building the 'Mopidy-MPRIS'
+pkgname=mopidy-mpris
+version=1.3.1
+revision=1
+noarch=yes
+build_style="python-module"
+pycompile_module="mopidy-mpris"
```
Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: There is no _ in this line. There should be. Please see my original comment.

**Comment by Vaelatern** [comment]
https://gist.github.com/2244363b735394ca57eb

The fact that it's `python2.7/site-packages/`**`mopidy_mpris`** means the `pycompile_module="mopidy_mpris"`

**Comment by Pizza-Boy** [comment]
Ah alright. 

I'm away from my computer at the moment but I'll fix the line in a bit.

**Comment by Vaelatern** [comment]
Does this work on your system? How did you get past the needing a `mopidy.desktop` file?

**Comment by Pizza-Boy** [comment]
Oh wow, didn't realize that the mopidy install doesn't install a .desktop file. 

Could we maybe pull the .desktop file from the mopidy source during installation of this package and `vmove` it to the right folder? Or is there a specific reason that the .desktop file isn't installed?

**Comment by Vaelatern** [comment]
9f8c1d5 now mopidy has the desktop file

**Comment by Pizza-Boy** [comment]
Great! is there any more editing required for the template?

File: srcpkgs/mopidy-mpris/template [hunk 6]
```diff
@@ -0,0 +1,16 @@
+#Template for building the 'Mopidy-MPRIS'
+pkgname=mopidy-mpris
+version=1.3.1
+revision=1
+noarch=yes
+build_style="python-module"
+pycompile_module="mopidy_mpris"
+hostmakedepends="python-setuptools"
+depends="python-dbus mopidy"
```
Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: A comment above this, noting: `XXX: libindicate bindings for Python`

Comment (Vaelatern) [inline srcpkgs/mopidy-mpris/template]: Have you actually tested this package?

**Comment by Pizza-Boy** [comment]
Sorry about the delay. I haven't had time to test the package on my system but I should be able to tonight.

**Comment by Pizza-Boy** [comment]
I'm actually having trouble getting mopidy to work on my computer; I keep getting seg faults (which seems to be unrelated to mopidy-mpris).

**Comment by allan** [comment]
There is an update to mopidy 2.0.0 coming (see #3945).
I wouldn't create extra packages for the 60+ mopidy extensions but instead use `pip search mopidy` as suggested in the [docs](https://docs.mopidy.com/en/latest/installation/source/#installing-extensions).

**Comment by Pizza-Boy** [comment]
Alright.
If the preferred installation for mopidy extensions is through `pip` then feel free to close this request.

**Comment by Vaelatern** [comment]
Considering the improved installation message with mopidy, this PR can be closed in favor of `pip install`ing any necessary mopidy components.

**Pull request closed without merging**